blueprints/ember-bootstrap/wizard.js

```diff
@@ -0,0 +1,162 @@
+const chalk = require('chalk');
+const path = require('path');
+const fg = require('fast-glob');
+
+const PREPROCESSORS = ['none', 'sass', 'less'];
+const BS_VERSIONS = [3, 4];
+
+const GUARD = {
+  customInstall(config, answers) {
+    return answers.customInstall;
+  },
+  bs3(config, answers) {
+    return config.bootstrapVersion === 3 || answers.bootstrapVersion === 3;
+  },
+  noBS(config /*, answers*/) {
+    return !config.bootstrapVersion;
+  },
+  noPreprocessor(config, answers) {
+    return (
+      !config.preprocessor ||
+      config.preprocessor === 'none' ||
+      answers.preprocessor === 'none'
+    );
+  }
+};
+
+function buildGuards(config) {
+  return (...guards) => answers => guards.every(g => g(config, answers));
+}
+
+function defaults(answers) {
+  return {
+    bootstrapVersion: 3,
+    preprocessor: 'none',
+    importBootstrapTheme: false,
+    importBootstrapCSS: answers.preprocessor === 'none',
+    importBootstrapFont: answers.bootstrapVersion === 3,
+    insertEmberWormholeElementToDom: true,
+    whitelist: []
+  };
+}
+
+function getDefault(config, key, fallback) {
+  if (config.hasOwnProperty(key)) {
+    return config[key];
+  }
+  return fallback;
+}
+
+function getComponents() {
+  let componentsPath = path.resolve(
+    __dirname,
+    '..',
+    '..',
+    'app',
+    'components',
+    '*.js'
+  );
+  let files = fg.sync(componentsPath);
+  return files.map(file => path.basename(file, '.js'));
```

Comment:
There is just one special case with `bs-contextualhelp` here, which `bs-popover` and `bs-tooltip` extend from, but which itself is not a public component users should see. So it should also not showup in this list, which I guess (haven't tried) is the case now? 

Maybe we have to mark it as private, e.g. by renaming to `-bs-contextulahelp` and filtering out all `-` prefixed components here?

Reply:
Never mind, as I was playing with, found that this is not an issue at all, since `bs-contextualhelp` is not epxorted in the `app` tree... 🤦‍♂️